examples/00-fluent/exhaust_system.py

```diff
@@ -46,6 +44,7 @@
 # Perform required imports, which includes downloading and importing
 # the geometry file.
 
+# sphinx_gallery_thumbnail_path = '_static/exhaust_system.png'
```

Comment:
Thumbnail specification needs to be in a code block (also mentioned in the [sphinx-gallery documentation](https://sphinx-gallery.github.io/stable/configuration.html#providing-thumbnail)), otherwise the command will show up in the webpage to users, as it was doing before